plugins/Events/API.php

```diff
@@ -213,20 +217,4 @@ public function getCategoryFromNameId($idSite, $period, $date, $idSubtable, $seg
     {
         return $this->getDataTable(__FUNCTION__, $idSite, $period, $date, $segment, $expanded = false, $idSubtable);
     }
-
-    /**
-     * @param DataTable $dataTable
-     */
-    protected function filterDataTable($dataTable)
-    {
-        $dataTable->filter('Sort', array(Metrics::INDEX_NB_VISITS));
```

Comment:
* Sort was not needed => Big performance win
* The other two queued filters were already assigned and not needed at all